api/src/foreign.rs

```diff
@@ -462,11 +471,23 @@ macro_rules! doctest_helper_setup_doc_env_foreign {
 			.unwrap();
 		let mut wallet_config = WalletConfig::default();
 		wallet_config.data_file_dir = dir.to_owned();
-		let pw = "";
+		let pw = ZeroingString::from("");
```

Comment:
it would be cool to see a lazy static for this. C# for example has `string.Empty` etc

Reply:
Think this is okay, it's just test setup